wolfssl/openssl/hmac.h

```diff
@@ -35,9 +35,17 @@
 #include "prefix_hmac.h"
 #endif
 
+#include <wolfssl/wolfcrypt/hmac.h>
+
+typedef struct WOLFSSL_HMAC_CTX {
+    Hmac hmac;
+    int  type;
+    word32  save_ipad[WC_HMAC_BLOCK_SIZE  / sizeof(word32)];  /* same block size all*/
+    word32  save_opad[WC_HMAC_BLOCK_SIZE  / sizeof(word32)];
+} WOLFSSL_HMAC_CTX;
+
```

Comment:
Don't have includes below structure declaration.

Reply:
This is necessary because the following includes need to have this struct defined. Moving this after the include results in:
```
In file included from /usr/local/include/wolfssl/openssl/objects.h:30,
                 from /usr/local/include/wolfssl/openssl/evp.h:954,
                 from /usr/local/include/wolfssl/openssl/hmac.h:41,
                 from hmac.c:12:
/usr/local/include/wolfssl/ssl.h:4233:35: error: unknown type name ‘WOLFSSL_HMAC_CTX’; did you mean ‘WOLFSSL_SHA_CTX’?
 4233 |     WOLFSSL_EVP_CIPHER_CTX *ectx, WOLFSSL_HMAC_CTX *hctx, int enc);
      |                                   ^~~~~~~~~~~~~~~~
      |                                   WOLFSSL_SHA_CTX
/usr/local/include/wolfssl/ssl.h:4234:69: error: unknown type name ‘ticketCompatCb’
 4234 | WOLFSSL_API int wolfSSL_CTX_set_tlsext_ticket_key_cb(WOLFSSL_CTX *, ticketCompatCb);
      |                                                                     ^~~~~~~~~~~~~~
```